### PR TITLE
Add translation of missing reflection class methods

### PR DESCRIPTION
**Pull request by Fan2Shrek**
Here is the translation of the missing `ReflectionClass` methods

- `ReflectionClass::isEnum()`
- `ReflectionClass::isReadOnly()`

### CODE REVIEW

File: reference/reflection/reflectionclass/isenum.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Renvoie &true; si la classe est une <link linkend="language.enumerations">énumération</link>, &false; sinon.
```
Comment (Girgias) [inline reference/reflection/reflectionclass/isenum.xml:28]:
```suggestion
   Renvoie &true; si la classe est une <link linkend="language.enumerations">énumération</link>, sinon &false;.
```

Plutôt non?

File: reference/reflection/reflectionclass/isreadonly.xml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   &true; si la classe est en lecture seule, &false; sinon.
```
Comment (Girgias) [inline reference/reflection/reflectionclass/isreadonly.xml:28]: Ditto

File: reference/reflection/reflectionclass/isenum.xml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: ec2fe9a592f794978114ef5021db9f1d00c2e05d Maintainer: Fan2Shrek Status: ready -->
```
Comment (Girgias) [inline reference/reflection/reflectionclass/isenum.xml:2]:
```suggestion
<!-- EN-Revision: ec2fe9a592f794978114ef5021db9f1d00c2e05d Maintainer: Fan2Shrek Status: ready -->
<!-- Reviewed: yes -->
```

Il manque le reviewed tag

**Review by Girgias — CHANGES_REQUESTED** [review]
Some minor nits

**Comment by devnexen** [comment]
Thanks !

**Pull request merged**